contracts/cw-storage/src/state.rs

```diff
@@ -1,17 +1,23 @@
 use crate::error::BucketError;
 use crate::error::BucketError::EmptyName;
 use crate::msg::BucketLimits;
-use cosmwasm_std::Uint128;
-use cw_storage_plus::Item;
+use cosmwasm_std::{Addr, Uint128};
+use cw_storage_plus::{Index, IndexList, IndexedMap, Item, Map, MultiIndex};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
+pub const DATA: Map<String, Vec<u8>> = Map::new("DATA");
+
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct Bucket {
     /// The name of the bucket.
     pub name: String,
```

Comment:
I would retain information about the owner of the bucket, I mean the address who created it.